test/fixtures/README.md

```diff
@@ -15,3 +15,7 @@ hello
 ## more > than
 
 <p><strong>raw content</strong></p>
+
+The following text includes a reference to an anchor that causes problems in EPUB documents.
+
+To remove this anti-pattern, we can replace `&&/2`, `||/2`, and `!/1` by `and/2`, `or/2`, and `not/1` respectively.
```

Comment:
Added this line to demonstrate that we're transforming the links to problematic anchors in EPUB files.